pkg/kubelet/cm/dra/state/state_checkpoint_test.go

```diff
@@ -124,9 +125,10 @@ func TestCheckpointStateStore(t *testing.T) {
 		Namespace:  "default",
 		PodUIDs:    sets.New("139cdb46-f989-4f17-9561-ca10cfb509a6"),
 		CdiDevices: []string{"example.com/example=cdi-example"},
+		ClassName:  "class-name",
 	}
 
-	expectedCheckpoint := `{"version":"v1","entries":[{"DriverName":"test-driver.cdi.k8s.io","ClaimUID":"067798be-454e-4be4-9047-1aa06aea63f7","ClaimName":"example","Namespace":"default","PodUIDs":{"139cdb46-f989-4f17-9561-ca10cfb509a6":{}},"CdiDevices":["example.com/example=cdi-example"]}],"checksum":2939981547}`
+	expectedCheckpoint := `{"version":"v1","entries":[{"DriverName":"test-driver.cdi.k8s.io","ClaimUID":"067798be-454e-4be4-9047-1aa06aea63f7","ClaimName":"example","Namespace":"default","PodUIDs":{"139cdb46-f989-4f17-9561-ca10cfb509a6":{}},"CdiDevices":["example.com/example=cdi-example"],"ClassName":"class-name"}],"checksum":2939981547}`
```

Comment:
can we add (here or in its proper place) a test with 2+ claims? It seems to me this should be allowed by the DRA API, either already or in the near future.

Reply:
done

Reply:
This comment is more relevant on:
https://github.com/kubernetes/kubernetes/pull/116513

I will add such a test

Reply:
Tests added in other PR